sdk/extensions/wcf/Microsoft.CoreWCF.Azure.StorageQueues/src/Microsoft.CoreWCF.Azure.StorageQueues.csproj

```
@@ -0,0 +1,44 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <RequiredTargetFrameworks>netstandard2.0</RequiredTargetFrameworks>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```

```suggestion
    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
```

If you're using the standard set or keeping the override on L4, no reason for a hardcode here.